daprdocs/content/en/reference/api/configuration_api.md

```diff
@@ -0,0 +1,87 @@
+---
+type: docs
+title: "Configuration API reference"
+linkTitle: "Configuration API"
+description: "Detailed documentation on the configuration API"
+weight: 700
+---
+
+## Component file
+
+A Dapr `configuration.yaml` component file has the following structure:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: configuration.<TYPE>
+  version: v1
+  metadata:
+  - name:<KEY>
+    value:<VALUE>
+```
+
+| Setting | Description                                                                            |
+| ------- |----------------------------------------------------------------------------------------|
+| `metadata.name` | The name of the configuration store.                                                   |
+| `spec/metadata` | An open key value pair metadata that allows a binding to define connection properties. |
+
+## Get Configuration
+
+This endpoint lets you get configuration from store.
+
+### HTTP Request
+
+```
+POST http://localhost:<daprPort>/v1.0/configuration/<storename>
```

Comment:
Should this not be 
```suggestion
POST http://localhost:<daprPort>/v1.0/configuration/<storename>/<key>
```

Reply:
Yeah, it should be GEY method. I 'll fix it.